src/vs/code/electron-main/window.ts

```diff
@@ -373,6 +373,19 @@ export class VSCodeWindow {
 			}
 		});
 
+		// Swipe command support (macOS)
+		this._win.on('swipe', (e, cmd) => {
```

Comment:
@seivan can we reuse this code with the `on('app-command')` handler? Seems duplicate.